pimd/pim_msdp.h

```diff
@@ -388,6 +398,12 @@ static inline bool pim_msdp_peer_config_write(struct vty *vty,
 static inline void pim_msdp_shutdown(struct pim_instance *pim, bool state)
 {
 }
+
+static inline void pim_msdp_originator_id(struct pim_instance *pim, const struct prefix *group,
+					  struct in_addr *originator_id)
+{
+	originator_id->s_addr = INADDR_ANY;
```

Comment:
PIMv6 is not aware of the originator id?

Reply:
MSDP is IPv4 only, PIMv6 uses embedded RP instead.

See my clean up PR ( #17636 )

Reply:
Thanks.